mender/templates/api-gateway-configmap.yaml

```diff
@@ -1,3 +1,5 @@
+{{ $entrypoint := ternary "https" "http" (.Values.api_gateway.env.SSL) }}
```

Comment:
```suggestion
{{ $scheme := ternary "https" "http" (.Values.api_gateway.env.SSL) }}
```
with this single variable might refer to everywhere.

Reply:
Comments accepted and addressed in commit 53d7e06.